posts/2023-06-21-data-protection-compliance-with-distributed-ledger-erasure.md

```diff
@@ -0,0 +1,241 @@
+
+Organizations acting as data controllers and using third party platforms as data processors must consider the platforms
+ability to support timely and permanent data erasure in order to maintain compliance with these regulations.
+
+## Data erasure in distributed ledgers
+
+In many traditional systems, permanent data erasure is trivial. It is often as simple as issuing a SQL query to delete rows
+or columns from a transactional database, or deleting a file from a file system or object store.
+
+However, in distributed ledger systems the concept of permanent and irrevocable erasure is fundamentally
+at odds with the design of such systems. Ledgers are by nature immutable. 
+
+How can organizations stay in compliance when storing personal information in an immutable, append-only ledger?
+
+A typical approach often involves client-side "crypto-shredding" where all personal information is encrypted
+using an encryption key on the client before it is persisted to the ledger. Then if a user requests for information to be deleted, 
+the data controller simply needs to delete the encryption key.
+
+This approach, while conceptually simple, involves significant complexity in practice. For example, key management
+becomes a major constraint. To support granular erasure of data on a field or person level may require 
+the storage and management of large numbers of encryption keys. Secure storage of critical encryption keys in an external system
+is a significant challenge in its own right, and retrieving the key and decrypting on read queries may have real performance impacts in practice.
+
+## Vendia's approach to data erasure
+
+Vendia Share provides an immutable, tamper-proof, and cryptographically verifiable distributed ledger.
+
+However, unlike traditional blockchain solutions, Vendia Share includes data protection
+capabilities built directly into the underlying ledger technology. 
+
+Vendia's distributed ledger supports [fine-grained access control (redaction)](https://www.vendia.com/blog/data-protection-approaches)
+as well as a novel approach to permanent and irrevocable data erasure.
+
+
+### Vaulted tokenization
+Vendia uses a _Vaulted Tokenization_ approach to support granular erasure while retaining the tamper-proof, verifiable, 
+and immutable properties of the distributed ledger.
+
+With this approach, user data stored in the ledger (mutations) are stored on-chain in a 
+tokenized form whereby erasable fields are replaced with cryptographically hashed values (tokens).
+
+Plaintext field values are stored in a chain-adjacent vault. Queries to the ledger will automatically detokenize the mutations, replacing the tokens with the corresponding plaintext value for fields that have not been erased.
+
+When erased, plaintext values are permanently and irrevocably removed from the vault but 
+historical data in the ledger itself is never modified. 
+Values stored in the vault can be removed but never modified.
```

Comment:
Umm.. Though I understand what this means, I just feel that removal is a subset of modified because it's gone. If we partially remove something then it's modified right?